src/Resources/ResourceManager/SdkClient/ProviderFeatureClient.cs

```diff
@@ -155,6 +155,47 @@ public PSProviderFeature UnregisterProviderFeature(string providerName, string f
             return this.FeaturesManagementClient.Features.Unregister(providerName, featureName).ToPSProviderFeature();
         }
 
+        /// <summary>
+        /// Lists the feature registrations on the current subscription
+        /// </summary>
+        /// <param name="providerName">The name of the resource provider</param>
+        /// <param name="featureName">The name of the feature</param>
+        public SubscriptionFeatureRegistration[] ListFeatureRegistrations(string providerName = null, string featureName = null)
```

Comment:
Please wrap .NET SDK type SubscriptionFeatureRegistration, refer to https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#returning-wrapped-sdk-types